job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/BuildInDockerContext.groovy

```diff
@@ -10,10 +12,12 @@ class BuildInDockerContext implements Context {
     List<Node> volumes = []
     boolean privilegedMode
     boolean verbose
+    boolean forcePull
     String userGroup
     String startCommand = '/bin/cat'
 
-    BuildInDockerContext() {
```

Comment:
You need to find all calls to the constructor and add the `jobManagement` parameter. You can build and test the plugin locally by running `./gradlew clean build` to check for problems before pushing changes.